src/Tickets/Commerce/Gateways/Stripe/Client.php

```diff
@@ -243,6 +345,17 @@ public function get_payment_intent_transient() {
 		return get_transient( $this->get_payment_intent_transient_name() );
 	}
 
+	/**
+	 * Delete the payment intent transient
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function delete_payment_intent_transient() {
+		return delete_transient( $this->get_payment_intent_transient_name() );
+	}
```

Comment:
That is totally fine for now, but I am still confused why we are storing payment Intents as transients and not as an order. In my eyes the moment we have a payment intent we need to create an order so that there is a log of that on the customers side.

I know this is too baked in now, but I would love to see after launch us coming back to this and fixing so that there is less transient usage and more around orders.